src/backend/marsha/core/api/base.py

```diff
@@ -153,6 +156,30 @@ def recording_slices_state(request):
     return Response(video.get_recording_slices_state())
 
 
+@api_view(["GET"])
+def get_frontend_configuration(request):
+    """View handling GET request to get the frontend configuration.
+
+    Parameters
+    ----------
+    request : Type[django.http.request.HttpRequest]
+        The request on the API endpoint.
+
+    Returns
+    -------
+    Type[rest_framework.response.Response]
+        HttpResponse containing the frontend configuration.
+
+    """
+    return JsonResponse(
+        {
+            "environment": settings.ENVIRONMENT,
+            "release": settings.RELEASE,
+            "sentry_dsn": settings.SENTRY_DSN if switch_is_active(SENTRY) else None,
```

Comment:
```suggestion
            "sentry": {
                "environment": settings.ENVIRONMENT,
                "release": settings.RELEASE,
                "sentry_dsn": settings.SENTRY_DSN if switch_is_active(SENTRY) else None,
            },
```

I think it should be under a sentry key. This endpoint is generic for other configuration, we should easily identify them.

Reply:
ok, I thought environment and release, even if used only for sentry, were more global.
I'll change it as you said.